### PR TITLE
Implement GitHub actions workflows

### PR DESCRIPTION
**Pull request by Normo**
* migrate from GitLab CI/CD to Github Actions

### CODE REVIEW

File: .github/workflows/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# SPDX-FileCopyrightText: 2022 Helmholtz Centre for Environmental Research (UFZ)
```
Comment (tobiashuste) [inline .github/workflows/dependabot.yml:1]: This file needs to moved one level up, isn't it?

**Review by tobiashuste — CHANGES_REQUESTED** [review]
The changelog entry

File: .github/workflows/dependabot.yml (hunk 1 above)
Comment (Normo) [reply .github/workflows/dependabot.yml]: Yes. This also explains the failed dependabot check. Moved it one level up. Thanks for the hint.

**Review by tobiashuste — APPROVED** [review]
Thanks, for transferring this project :shipit:

**Pull request merged**